src/index.css

```diff
@@ -17,3 +17,47 @@
 .error {
   color: red;
 }
+
```

Comment:
We use milligram CSS and want to keep additional styling as minimal as possible. Please use [buttons](http://milligram.io/#buttons). This is because we want to later easily change this styling (the lower the classes, the easier it is to change) for each year's theme